### ISSUE TITLE
Linking lifecycle badges to stage definitions

### ISSUE THREAD

**Issue by jsta** [r-lib/usethis#193]
The lifecycle badges are great but unlike the repostatus badges they link to standalone svg images rather than stage definitions. I think linking the lifecycle badges to their definitions would be a nice improvement.

**Comment by jennybc** [comment]
@hadley @batpigandme 

Where should be put this info?

https://github.com/r-lib/usethis/blob/cafc90eb73ed412ff589444321f09f462bbdf384/R/badge.R#L41-L55

I plan to make a very basic pkgdown website for usethis soon but am not sure that is the right definitive place for the badge explanations. Is it? Or somewhere below tidyverse.org?

**Comment by hadley** [comment]
Once it exists, I'm imagining it will be part of the lifecycle package website (e.g. lifecycle.r-lib.org). We could certainly put it on tidyverse.org in the meantime.

**Comment by batpigandme** [comment]
Agreed. I think it'd be a super useful reference to have pretty much anywhere!

**Comment by jennybc** [comment]
@batpigandme will you make the necessary PRs on tidyverse.org and here?

**Comment by batpigandme** [comment]
Sure thing. Are the lifecycle badges only on GitHub repos at this point?

**Comment by jennybc** [comment]
> Are the lifecycle badges only on GitHub repos at this point?

AFAIK, yes

**Comment by jimhester** [comment]
FWIW it needs to change to something else than the svg, because rmarkdown / pandoc converts 

```
[![lifecycle](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://img.shields.io/badge/lifecycle-maturing-blue.svg)
```
To
```
<https://img.shields.io/badge/lifecycle-maturing-blue.svg>
```

If you try to use the badges currently generated in a README.Rmd.

**Comment by batpigandme** [comment]
@jimhester copy that, thanks for the heads up

**Comment by batpigandme** [comment]
^^ Just realized the above comment re. SVG might not actually be relevant to the webpage, which I changed to PNG…
